lib/Elastica/Multi/MultiBuilder.php

```diff
@@ -38,16 +38,15 @@ private function buildResultSet(Response $childResponse, BaseSearch $search)
      */
     private function buildResultSets(Response $response, $searches)
     {
-        $resultSets = array();
-
         $data = $response->getData();
-        if (!isset($data['responses']) || !is_array($data['responses'])) {
-            return $resultSets;
+        if (empty($data['responses']) || !is_array($data['responses'])) {
```

Comment:
I don't think these two are here the same. If `$data['response] == false` the first one would not catch it.

Overall as we look for an `array` to in the end it seems to be the same. Do we have unit tests here to make sure that is the case?